code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm

```diff
@@ -401,12 +401,19 @@
 			to_chat(owner, "<span class='warning'>There is a resin node in the way!</span>")
 		return FALSE
 
+	if(locate(/obj/effect/alien/resin/acidwell) in range(1,T))
```

Comment:
You may consider using the following instead of 1


"A Dist of 0 includes Center, the contents of Center (normally usr.contents), its location (normally the turf a mob is standing on), and any other contents of that location. A value of 1 extends the region to the neighboring squares on the map and so on. You can also use a rectangular box size using a text string such as "13x11". Both arguments are optional and may be passed in any order"

as the `1` makes a + shape.

Reply:
See if "3x3" works. if you don't want traps set up diagonally from each other.

Reply:
So yea, I tested the range(1,T) and it includes diagonals.